lib/mixins/property-effects.html

```diff
@@ -2530,7 +2533,9 @@
           // Attribute or property
           let origName = name;
           let kind = 'property';
-          if (name[name.length-1] == '$') {
+          if (capitalAttributeRegex.test(name)) {
```

Comment:
Let's add a comment in the code here explaining why we're doing this. The only way we see a capital letter here is if the attr has a capital letter in it per spec. In this case, to make sure this binding works, we go ahead and make the binding to the attribute.

We should also add something simplified about this to the doc.

Reply:
@sorvell Done :+1: